src/ip-fragment.c

```diff
@@ -158,15 +152,14 @@ static inline int do_reassemble(struct pg_ip_fragment_state *state,
 	return ret;
 }
 
-static inline bool should_be_fragmented(struct rte_mbuf const * const pkt,
```

Comment:
humm, if you have to remove a const, maybe the error come from pg_utils_get_ether_type